src/components/AppHeader/AppHeader.js

```diff
@@ -47,11 +55,19 @@ const AppHeader = ({
 		<Menu mode="horizontal">
 			<Menu.Item key="back" className={noBorder} style={{ padding: 0 }}>
 				<Link to="/">
-					<Icon type="arrow-left" />
+					<Button css={backIcon} shape="circle" icon="arrow-left" />
 				</Link>
 			</Menu.Item>
 			<Menu.Item key="1" className={noBorder}>
-				<span>{currentApp || 'Loading...'}</span>
+				{appOwner
```

Comment:
How will this work with App Switcher?

![image](https://user-images.githubusercontent.com/9116042/47722689-ff2b1580-dc78-11e8-85fc-af9421c39cc2.png)

Reply:
We can show the owner on hover and on click we can show apps list

Reply:
I feel it'd be better if we only show the tooltip on hover at the share icon.

Reply:
Yeah that can be done. Will fix this